pkg/steps/new_e2e_test.go

```diff
@@ -94,8 +94,8 @@ func TestGenPods(t *testing.T) {
 			}},
 		},
 	}}
-	step := newNewE2ETestStep(config, params, nil, &jobSpec, nil)
-	ret, err := step.genPods(steps)
+	step := newNewE2ETestStep(config, params, nil, "", &jobSpec, nil)
```

Comment:
How come adding artifacts didn't change any test output?

Reply:
Because the step doesn't have an `ArtifactDir`.